src/core/agent/services/signifyNotificationService.ts

```diff
@@ -190,7 +202,19 @@ class SignifyNotificationService extends AgentService {
     if (event.a.r === NotificationRoute.MultiSigIcp) {
       const multisigNotification = await this.props.signifyClient
         .groups()
-        .getRequest(event.a.d);
+        .getRequest(event.a.d)
+        .catch((error) => {
+          const errorStack = (error as Error).stack as string;
+          // If the error is failed to fetch with signify, we retry until the connection is secured
+          if (
+            /404 not found/gi.test(errorStack) &&
```

Comment:
I think `404` is enough in case it slightly changes

Reply:
I'm afraid it can be duplicated with other texts in the error, like the arguments we pass to the method. We can get the status by using `error.stack.split("-")[1]`. Here is an example of an error from signify: 
`Error: HTTP GET /multisig/request/abc404 - 404  - {"title": "404 Not Found", "description": "no multisig request with said=abc404 found"}
    at SignifyClient.fetch (webpack-internal:///./node_modules/signify-ts/dist/signify-ts.mjs:7674:13)
    at async Groups.getRequest (webpack-internal:///./node_modules/signify-ts/dist/signify-ts.mjs:7204:17)
    at async MultiSigService.test (webpack-internal:///./src/core/agent/services/multiSigService.ts:545:28)
    at async test (webpack-internal:///./src/ui/pages/Chat/Chat.tsx:32:9)`